### ISSUE TITLE
Issue of striped lines appearing in photos.

### ISSUE THREAD

**Issue by kitcheng** [will127534/OneInchEye#22]
Thank you very much to the developers; this is an amazing project.

When I take photos at the highest resolution (5568x3664), there are striped lines at the edges of the photos. Is this a known issue that is considered normal, or is it abnormal and potentially related to factors such as software or cable connections?

![test_0000](https://github.com/user-attachments/assets/00676c08-0725-4d27-afa6-0fce8c3bebad)

**Comment by will127534** [comment]
Yes it is normal, those are extra pixels/black pixels at the edge that can be cropped out.
Specifically, the issue is that for some reason I can not cropped out those at the driver level, if I set the resolution to the supposedly correct sizes on RPI5 it will causes stride unmatching.

**Comment by kitcheng** [comment]
Alright, I understand. I'll crop it at the application layer. Thanks again.
